plugins/inventory/now.py

```diff
@@ -616,6 +634,8 @@ def _get_instance_from_env(self):
             password=os.getenv("SN_PASSWORD"),
             client_id=os.getenv("SN_CLIENT_ID"),
             client_secret=os.getenv("SN_SECRET_ID"),
+            refresh_token=os.getenv("SN_REFRESH_TOKEN"),
+            grant_type=os.getenv("SN_GRANT_TYPE"),
```

Comment:
not needed, should already be provided by `get_option` as it is defined in documentation already

Reply:
not even sure why this is here, it is not called from anywhere in this plugin